doc/source/serve/deployment-graph/control_flow_based_on_user_inputs.md

```diff
@@ -13,6 +13,24 @@ The example shows how to user inputs to control the graph flow
    :language: python
 ```
 
+```{note}
+1. The dag.execute() take arbitrary number of arguments, and internally the graph will access inputs by index (e.g. user_input[0], user_input[1]).
```

Comment:
"and internally we implemented data objects to facilitate accessing by index or key"